src/components/InstallTabs/index.tsx

```diff
@@ -83,12 +83,11 @@ const InstallTabs = (): JSX.Element => {
       </TabPanel>
       <TabPanel>
         <div>
-          <code className="install__text">
-            <div className="install__text__line">
-              <span className="install__text__no-select">$</span>choco install
-              nodejs-lts
-            </div>
-          </code>
+          <ShellBox textToCopy="choco install nodejs-lts">
+            <span className="install__text__no-select">$</span>
+            <span className="install__text__command">choco </span>install
```

Comment:
The nvm instructions include steps to install nvm, but there are no steps here to install chocolatey

Reply:
If you know of a bash way of installing chocolatey, I'll include it. I didn't see one here. https://chocolatey.org/install or https://chocolatey.org/docs/installation
@saulonunesdev any ideas on what should be include for how to install chocolatey?